daml-lf/spec/daml-lf-1.rst

```diff
@@ -79,12 +79,13 @@ mark lines within inference rules with annotations of the form
 ``[DAML-LF < x.y]`` and ``[DAML-LF ≥ x.y]`` to make the respective
 line conditional upon the DAML-LF version.
 
-Below, we list the versions of DAML-LF that a DAML-LF engine
-compliant with the present specification must handle.  The list comes
-with a brief description of the changes, and some links to help
-unfamiliar readers learn about the features involved in the change.
-One can refer also to the `Serialization` section which is
-particularly concerned about versioning and backward compatibility.
+Below, we list the versions of DAML-LF major version 1 that a DAML-LF
```

Comment:
Below, we list the full versions of DAML-LF 1.x that a DAML-LF engine compliant with the present specification must handle...

Reply:
0074aecea2